tests/synced_batchnorm/two_gpu_unit_test.py

```diff
@@ -92,6 +92,8 @@ def compare(desc, inp1, inp2, error):
 grad_bn = grad_output_t.clone().detach()
 out_bn = bn(inp_bn)
 out_bn.backward(grad_bn)
+for param in bn.parameters():
+    param.grad = param.grad / args.world_size
```

Comment:
Maybe add a comment here explaining that the division by world_size is included to make the single-process control case equivalent to the averaging over processes that is done by DDP.